frontend/src/metabase/admin/settings/components/widgets/SettingToggle.jsx

```diff
@@ -4,13 +4,17 @@ import { t } from "ttag";
 import Toggle from "metabase/core/components/Toggle";
 import Tooltip from "metabase/core/components/Tooltip";
 
-const SettingToggle = ({ setting, onChange, disabled, tooltip }) => {
+const SettingToggle = ({ id, setting, onChange, disabled, tooltip }) => {
   const value = setting.value == null ? setting.default : setting.value;
   const on = value === true || value === "true";
   return (
     <div className="flex align-center pt1">
       <Tooltip tooltip={tooltip} isEnabled={!!tooltip}>
-        <Toggle value={on} onChange={!disabled ? () => onChange(!on) : null} />
+        <Toggle
+          id={id}
+          value={on}
+          onChange={!disabled ? () => onChange(!on) : null}
```

Comment:
Ah good call!

Reply:
@kulyk I tested it, even though it pass `disabled` to the `input`, we don't use that information to style the toggle that looks disabled. Do you think I should adjust the css to make it looks disabled too?

#### How it currently looks (the same as non disabled)
<img width="592" alt="image" src="https://user-images.githubusercontent.com/1937582/218155989-7fece5b4-e7b5-4169-bee5-6cfa39e31429.png">

@kdoh Do we have a disabled toggle design? It could be easy to add, but I'm not sure how we want to make them look.

Reply:
I wonder if the toggle's look changes, if we _, don't_ pass the `onChange` prop? Overall I think the `disabled` prop is a more common way to disable inputs vs. not passing `onChange`